src/routes/api/category-routes.js

```diff
@@ -0,0 +1,99 @@
+const router = require("express").Router();
+const { Category, Product } = require("../../models");
+
+// The `/api/categories` endpoint
+
+router.get("/", async (req, res) => {
```

Comment:
Although this is okay, it would be great if you could separate these controller functions into another `controllers` file to ensure the routes are kept clear and easy to follow.